evap/evaluation/templates/questionnaires_widget.html

```diff
@@ -27,7 +27,7 @@
             </ul>
             {% endspaceless %}">
             <input class="form-check-input" id="{{ choice.id_for_label }}" name="{{ choice.data.name }}" type="checkbox" value="{{ choice.data.value }}"
-                autocomplete="off"{% if choice.data.selected %} checked{% endif %}
+                autocomplete="off"{% if choice.data.selected or questionnaire.is_locked and not manager and choice.data.value in field.initial %} checked{% endif %}
```

Comment:
This change is necessary to keep locked questionnaires checked even when the form was reloaded with errors, right?
I do not know whether it is possible, but could the `choice.data.selected` part be modified by the form? Thinking this further, is it possible to include a `choice.data.is_locked` attribute (or similar)? Just thinking loud, this isn’t blocking

Reply:
right, this is needed so that locked questionnaires are checked on form errors, your thoughts are interesting but i'd prefer you investigating them later on :)